tailwind.config.js

```diff
@@ -179,31 +181,9 @@ module.exports = {
       2: 2,
     },
     fontFamily: {
-      system: [
-        'ui-sans-serif',
-        'system-ui',
-        '-apple-system',
-        'BlinkMacSystemFont',
-        '"Segoe UI"',
-        'Roboto',
-        '"Helvetica Neue"',
-        'Arial',
-        '"Noto Sans"',
-        '"Oxygen"',
-        '"Cantarell"',
-        '"Fira Sans"',
-        '"Droid Sans"',
-        'sans-serif',
-        '"Apple Color Emoji"',
-        '"Segoe UI Emoji"',
-        '"Segoe UI Symbol"',
-        '"Noto Color Emoji"',
-      ],
-      heading: ['"Inter"', 'sans-serif'],
       mono: ['"JetBrains Mono"', 'monospace'],
-      sans: ['"Inter"', 'sans-serif'],
-      serif: ['Times New Roman'],
```

Comment:
We could potentially let `serif` go, I don't know if we use it anywhere in the app.